Add DigitsSession#getUserId() to avoid ambiguity between session ID a…

daisy1754 opened the pull request:
…nd user ID (currently we use same value). Also update caller of getId() to use getUserId() if caller want user ID

efrohnhoefer commented:
Thanks for the pull request however changing the public interface is a breaking change. Can we just deprecate the old usage? Please make sure to add a @Deprecated comment to the jdoc.

daisy1754 commented:
Thanks for quick reply. Note I added a new method, not breaking existing APIs. 
I agree adding *@Deprecated* to existing method is a good idea, so I added a such commit.

mikebonnell commented:
Thank you for posting this PR. As the Digits SDK is evolving into [Firebase phone authentication](https://github.com/firebase/FirebaseUI-Android ) and being deprecated at the end of September, we don't plan on reviewing this so I'm marking it as closed.

Firebase phone authentication includes the best of Digits plus more power, reliability and new features to drive user growth. [Learn more here!](http://get.digits.com/blog/introducing-firebase-phone-authentication
)
